src/main/java/io/github/perrymant/moneymaker/Logger.java

```diff
@@ -2,8 +2,19 @@
 
 class Logger {
     void info(final String str) {
-        System.out.println("Date\t\t| Transaction Type\t| Amount\t| Description\t\t| Balance\t|");
-        System.out.println("=============================================================================");
+        String logo =
+                "  /$$      /$$   Welcome to ...                        /$$      /$$           /$$                           \n" +
+                " | $$$    /$$$                                        | $$$    /$$$          | $$                           \n" +
+                " | $$$$  /$$$$  /$$$$$$  /$$$$$$$   /$$$$$$  /$$   /$$| $$$$  /$$$$  /$$$$$$ | $$   /$$  /$$$$$$   /$$$$$$  \n" +
+                " | $$ $$/$$ $$ /$$__  $$| $$__  $$ /$$__  $$| $$  | $$| $$ $$/$$ $$ |____  $$| $$  /$$/ /$$__  $$ /$$__  $$ \n" +
+                " | $$  $$$| $$| $$  \\ $$| $$  \\ $$| $$$$$$$$| $$  | $$| $$  $$$| $$  /$$$$$$$| $$$$$$/ | $$$$$$$$| $$  \\__/ \n" +
+                " | $$\\  $ | $$| $$  | $$| $$  | $$| $$_____/| $$  | $$| $$\\  $ | $$ /$$__  $$| $$_  $$ | $$_____/| $$       \n" +
+                " | $$ \\/  | $$|  $$$$$$/| $$  | $$|  $$$$$$$|  $$$$$$$| $$ \\/  | $$|  $$$$$$$| $$ \\  $$|  $$$$$$$| $$       \n" +
+                " |__/     |__/ \\______/ |__/  |__/ \\_______/ \\____  $$|__/     |__/ \\_______/|__/  \\__/ \\_______/|__/       \n" +
+                "                                             /$$  | $$                                                      \n" +
+                "                                            |  $$$$$$/     ... Gold don't get old                           \n" +
+                "                                             \\______/                                                       ";
+        System.out.println(logo);
```

Comment:
Do we always want to print the logo whenever we want to log something in our application? I'm guessing no - maybe we should only print the logo once on startup?